src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -50,9 +50,9 @@
     <Reference Include="FontAwesome.WPF">
       <HintPath>..\..\extern\FontAwesome\FontAwesome.WPF.dll</HintPath>
     </Reference>
-    <Reference Include="Greg, Version=1.0.5892.25330, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\extern\greg\Greg.dll</HintPath>
+    <Reference Include="Greg, Version=1.0.5950.12824, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Greg.1.0.5950.12824\lib\net45\Greg.dll</HintPath>
+      <Private>True</Private>
```

Comment:
If there are many projects that reference the binaries from Greg package then it's better to define a variable in CS.props file and use that path. So next time when we need to upgrade the package we don't need to modify all the projects, just update the CS.props file.